src/dbg/debugger.cpp

```diff
@@ -127,7 +128,7 @@ static DWORD WINAPI memMapThread(void* ptr)
             memMapThreadCounter = GetTickCount();
         }
 
-        Sleep(50);
+        Sleep(1000);

```

Comment:
This timer might be too long. It might be worth to change this to something similar as the `TaskThread` you implemented. The semantics I would prefer are like `2000ms` when idle and an immediate wake up as soon as an update is required.